assets/javascript/game.js

```diff
@@ -1,219 +1,95 @@
+  var audioElement1 = document.createElement('audio');
+  audioElement1.setAttribute('src', 'https://s3-us-west-1.amazonaws.com/week4sounds/sounds/register.WAV');
 
-			var crystal1 = 0;
-			var crystal2 = 0;
-			var crystal3 = 0;
-			var crystal4 = 0;
+  var audioElement2 = document.createElement('audio');
+  audioElement2.setAttribute('src', 'https://s3-us-west-1.amazonaws.com/week4sounds/sounds/register.mp3');
 
-			var audioElement1 = document.createElement("audio");
-      		audioElement1.setAttribute("src", "https://s3-us-west-1.amazonaws.com/week4sounds/sounds/register.WAV");
+  var song = document.createElement('audio');
+  song.setAttribute('src', 'https://s3-us-west-1.amazonaws.com/week4sounds/sounds/song.mp3');
 
-      		var audioElement2 = document.createElement("audio");
-      		audioElement2.setAttribute("src", "https://s3-us-west-1.amazonaws.com/week4sounds/sounds/register.mp3");
+  var wins = 0;
+  var losses = 0;
 
-      		var song = document.createElement("audio");
-      		song.setAttribute("src", "https://s3-us-west-1.amazonaws.com/week4sounds/sounds/song.mp3");
+  var counter;
+  var numberGuess;
 
+  var crystal1 = $( '#crystal1_span' );
+  var crystal2 = $( '#crystal2_span' );
+  var crystal3 = $( '#crystal3_span' );
+  var crystal4 = $( '#crystal4_span' );
 
-			// This function chooses the random number for each crystal. 
-			function CrystalNumber(crystalName) {
-				return Math.ceil(Math.random() * 12 - 1) + 1;
-			};
 
-			crystal1 = CrystalNumber(crystal1);
-			crystal2 = CrystalNumber(crystal2);
-			crystal3 = CrystalNumber(crystal3);
-			crystal4 = CrystalNumber(crystal4);
-			console.log(crystal1, crystal2, crystal3, crystal4);
+  $('#startButton').on('click', function(){
+    audioElement1.play();
 
-			var numberGuess = Math.floor(Math.random() * 120) + 19;
+    $( '#startButton' ).effect( 'puff' );
+    $( '#crystal1_span' ).effect( 'pulsate' );
+    $( '#crystal2_span' ).effect( 'pulsate' );
+    $( '#crystal3_span' ).effect( 'pulsate' );
+    $( '#crystal4_span' ).effect( 'pulsate' );
+  });
 
-			// Check to see if the numberGuess is working. 
-			console.log(numberGuess);
 
-			// This sends the number the player is trying to reach to the #numberToGuess div.
-			$("#numberToGuess").html(numberGuess);
+  $('.crystal-btn').on('click', function(){
+    $( this ).effect( 'pulsate' )
+    counter += $( this ).data( 'crystal-value' )
+    $( '#playerCounter' ).html( counter );
 
-			// This is the function that controls the game play. 
+    if (counter > numberGuess) {
+      gameLose();
+    } else if (counter === numberGuess) {
+      gameWin();
+    };
 
-			// song.play();
+  });
 
-			function game () {
+  function generateRandomNumber (min, max) {
+    return Math.floor(Math.random() * max) + min;
+  };
 
-				// These show the current score. 
-				$("#playerCounter").html(counter);
-				$("#playerWins").html(wins);
-				$("#playerLosses").html(losses);
+  function setRandomGameValues () {
```

Comment:
This assortment of game values needs to be generated and set into the html at the same time at multiple points throughout the game. You should try to be on the look out for repetitive code that you can factor out into a function similar to this.